metainfo.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/packaging/metadata/org.coolercontrol.CoolerControl.metainfo.xml b/packaging/metadata/org.coolercontrol.CoolerControl.metainfo.xml
+index b16da4e..5f298f8 100644
+--- a/packaging/metadata/org.coolercontrol.CoolerControl.metainfo.xml
++++ b/packaging/metadata/org.coolercontrol.CoolerControl.metainfo.xml
+@@ -20,6 +20,10 @@
+         <p>
+             It's an extension of liquidctl and hwmon with a focus on controlling cooling devices such as AIO coolers and fans.
+         </p>
++        <p>
++            Note: The coolercontrold system package is required to control and communicate with your hardware devices. 
```

Comment:
this is gonna work great out of the box.

Reply:
There are several other Flatpaks that work like this but there is also a history here.
This project went through a name and implementation change and was previously called Coolero and there is an outdated flatpak here: https://flathub.org/apps/org.coolero.Coolero

This project now consists of several packages which is why this note is here and the idea is essentially to replace the above flatpak with this alternative for the current implementation. 

If this is not appropriate or there is an issue with this, please say so. The GUI also starts and displays some helpful dialog about the needed systemd service if it isn't already running.

Reply:
two wrong don't make a right.